_data/tutorials.yaml

```diff
@@ -43,10 +43,10 @@ merging:
     kafka: enabled
 
 joining-stream-table:
-  title: "How to join a stream and a table together"
-  meta-description: "Learn to join a stream and a table together"
+  title: "How to join a stream and a lookup table together"
+  meta-description: "Learn to join a stream and a lookup table together"
```

Comment:
@awalther28 awesome touch to augment `meta-description` as well, since that text may sometimes be surfaced in Google search results, which in turn impacts SEO and CTR.

Reply:
@ybyzek Do you think I need to go further into detail throughout the KT about databases+lookups OR do you think these modifications are enough?

Reply:
LGTM!